src/region.js

```diff
@@ -86,7 +86,7 @@ const Region = MarionetteObject.extend({
 
   _attachView(view, {replaceElement} = {}) {
     const shouldTriggerAttach = !view._isAttached && isNodeAttached(this.el);
-    const shouldReplaceEl = !!replaceElement || !!this.getOption('replaceElement');
+    const shouldReplaceEl = replaceElement === true || !!this.getOption('replaceElement');
```

Comment:
Hahaha. This is what I get for trying to do a 'simple fix' from GitHub's IDE. Attempt #3 committed.